bundles/best_practices.rst

```diff
@@ -166,6 +166,70 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+    env:
+        global:
+            - TEST_COMMAND="./vendor/bin/simple-phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported PHP and Symfony version
+            - php: 7.2
+              env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="phpunit --coverage-text" SYMFONY_DEPRECATIONS_HELPER="weak"
+
+              # Test the latest stable release
+            - php: 7.0
+            - php: 7.1
+            - php: 7.2
+
+              # Test LTS versions
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:^2"
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:^3"
+
+              # Latest commit to master
+            - php: 7.2
+              env: DEPENDENCIES="dev"
+
+        allow_failures:
+              # Latest beta is allowed to fail.
+            - php: 7.2
+              env: DEPENDENCIES="dev"
+
+    before_install:
+        - composer require --no-update "symfony/phpunit-bridge:^3.3 || ^4"
+        - if [[ $COVERAGE != true ]]; then phpenv config-rm xdebug.ini || true; fi
+        - if [ "$DEPENDENCIES" = "minimum" ]; then COMPOSER_FLAGS="--prefer-stable --prefer-lowest"; fi;
+        - if [ "$DEPENDENCIES" = "dev" ]; then composer config minimum-stability dev; fi;
+        - if [[ $DEPENDENCIES == *"/"* ]]; then composer require --no-update $DEPENDENCIES; fi;
+
+    install:
```

Comment:
don't forget to add `vendor/bin/simple-phpunit install` here (as you did on httplug) to have the installation output be collapsed by default.

Reply:
Thanks!